Solutions/Corvus.Testing.AzureFunctions/Corvus/Testing/AzureFunctions/ProcessStartupException.cs

```diff
@@ -0,0 +1,38 @@
+﻿// <copyright file="ProcessStartupException.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Testing.AzureFunctions
+{
+    using System;
+
+    /// <summary>
+    /// Represents a failure in starting a process.
```

Comment:
The generality of this makes it feel out of place. I'm not convinced this library has any business defining such a general-purpose exception. (Although that may be more of a documentation/naming issue: it's not really all that general purpose in practice, because in most cases where a process fails to start, you're not going to have stdout and stderr.)